CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.0.8
+
+- ### Features
+
+  - **Support a variety of errors thrown from `getAssetFromKV` - [victoriabernard92], [issue/59] [pull/64]**
+
+    Support for throwing specific errors to allow the handler of `getAssetFromKV` to catch and differentiate them.
```

Comment:
```suggestion
   Previously, `getAssetFromKv` would throw the same error type if anything went wrong. Now it will throw different error types so that clients can catch and differentiate them.
```